gensim/models/keyedvectors.py

```diff
@@ -752,8 +745,13 @@ def wmdistance(self, document1, document2):
             If `pyemd <https://pypi.org/project/pyemd/>`_  isn't installed.
 
         """
-        if not PYEMD_EXT:
-            raise ImportError("Please install pyemd Python package to compute WMD.")
+
+        # If pyemd C extension is available, import it.
+        # If pyemd is attempted to be used, but isn't installed, ImportError will be raised in wmdistance
+        try:
+            from pyemd import emd
+        except ImportError:
+            logger.error('Pyemd not installed')
```

Comment:
Why this change from an exception to logging an error and continuing?

Reply:
My bad. I moved this piece of code from the top to this position because of a [failure](https://travis-ci.org/RaRe-Technologies/gensim/jobs/445388889) in the previous commit. I'll fix this.